advisor/src/test/kotlin/advisors/VulnerableCodeTest.kt

```diff
@@ -77,130 +77,102 @@ class VulnerableCodeTest : WordSpec({
 
     "VulnerableCode" should {
         "return vulnerability information" {
-            stubFor(
-                post(urlPathEqualTo("/api/packages/bulk_search"))
-                    .withRequestBody(equalToJson(packagesRequestJson, false, false))
-                    .withHeader("Content-Type", equalTo("application/json; charset=UTF-8"))
-                    .willReturn(
-                        aResponse().withStatus(200)
-                            .withBodyFile("response_packages.json")
-                    )
-            )
-
-            stubVulnerability("v1", "CVE-2014-8242", 42.0f)
-            stubVulnerability("v2", "CVE-2009-1382", 11.0f)
-            stubVulnerability("v3", "CVE-2019-CoV19", 77.0f)
-
+            wiremock.stubPackagesRequest("response_packages.json")
             val vulnerableCode = createVulnerableCode(wiremock)
-            val packagesToAdvise = resultFile().readValue<OrtResult>().getPackages(false).map { it.pkg }
+            val packagesToAdvise = inputPackages()
 
             val result = vulnerableCode.retrievePackageVulnerabilities(packagesToAdvise).mapKeys { it.key.id }
 
             result.shouldNotBeEmpty()
             result.keys should containExactlyInAnyOrder(idLang, idStruts)
 
             val langResults = result.getValue(idLang)
-            langResults shouldHaveSize(1)
+            langResults shouldHaveSize (1)
```

Comment:
Nit: Unintended change?

Reply:
If you wanted to use infix notation, you simply should have omitted the parentheses, or used `should haveSize(1)`. But such a change should go to a separate commit.